Updated readme and added mechanical design files (#1)
TU-Darmstadt-APQ/MOT_Photodiode: README.md

[![Build manufacturing files](https://github.com/TU-Darmstadt-APQ/MOT_Photodiode/actions/workflows/ci.yml/badge.svg)](https://github.com/TU-Darmstadt-APQ/MOT_Photodiode/actions/workflows/ci.yml)
![License: CERN-OHL-W-2.0](https://img.shields.io/github/license/TU-Darmstadt-APQ/MOT_Photodiode?color=blue)
# MOT Photodiode
This is a photodiode board with a transimpedance amplifier for [S-100](https://osioptoelectronics.com/media/pages/products/photodetectors/solderable-chip-photodiodes/s-100vl/1aa00dfa44-1675100439/solderable-chips.pdf) photodiodes used to probe the MOT beams at the QUIPS-C experiment.

> The photodiode is inserted on a slide-in unit ([slide_in_mount](photodiode_mount/slide_in_mount.ipt) in combination with [photo_plate_slide_in](photodiode_mount/photo_plate_slide_in.ipt)) which can be placed behind the MOT Beam Doublett colimators. Which itself is mounted to a flange on the vacuum chamber (viewport_mount assembly designed by Lars Pause). The photodiode board can be screwed on this holder with two polymer screws. 

Reviewer: ```suggestion
The photodiode is inserted into the laser beam on a slide-in unit ([slide_in_mount](photodiode_mount/slide_in_mount.ipt) in combination with the [photo_plate_slide_in](photodiode_mount/photo_plate_slide_in.ipt)) which can be placed behind the MOT Beam Doublett collimators. This assembly is mounted on a flange screwed to the vacuum chamber (viewport_mount assembly designed by Lars Pause). The photodiode board can be screwed on this holder with two polymer screws. 
```

> 

Reviewer: ```suggestion
```

- MINOR versions contain changes to the hardware that only affect the inner workings of the circuit, but otherwise the performance is unaffected.
- PATCH versions do not affect the schematics or invalidate older bill of materials. These changes may include updated components (to replace obsolete parts for example), an updated silkscreen, or fixed typos.

> 

Reviewer: ```suggestion
```

Reviewer: This line can also be dropped.

> Note: the original version by Tilman Preuschoff (2019) was only local available and not versioned on GitHub.

Reviewer: ```suggestion
```

Reviewer: This info should be part of the commit comment.
